Raider/GameModes/GameModeLateGame.hpp

```diff
@@ -13,46 +47,73 @@ class GameModeLateGame : public AbstractGameModeBase
 
         auto GameMode = static_cast<AFortGameModeAthena*>(GetWorld()->AuthorityGameMode);
 
-        GameMode->bSafeZoneActive = false;
-        GameMode->bSafeZonePaused = true;
+        GameMode->bSafeZoneActive = true;
+        GameMode->bSafeZonePaused = false;
     }
+    
 
     void InitializeGameplay()
     {
+
         auto GameState = static_cast<AFortGameStateAthena*>(GetWorld()->GameState);
         auto GameMode = static_cast<AFortGameModeAthena*>(GetWorld()->AuthorityGameMode);
         auto Aircraft = GameState->GetAircraft(0);
+        // GameState->GamePhase = EAthenaGamePhase::SafeZones;
+        // int SafeZoneConfigure[3] = { 1 }
+        // GameMode->SafeZoneLocations = GetRandomBattleBusLocation();
 
-        auto SafeZoneCenter = FVector({ 150, 150, 150 }); /* TODO: Array of Locations where late game could take place in. (POIs) */
-
-        if (GameMode->SafeZoneIndicator)
-        {
-            GameMode->SafeZoneIndicator->bPaused = true;
-            GameMode->SafeZoneIndicator->SetSafeZoneRadiusAndCenter(4000, SafeZoneCenter);
-        }
-        else
-        {
-            LOG_ERROR("SafeZoneIndicator was NULL! Can't set Radius and Center.");
-        }
+        auto SafeZoneCenter = GetRandomBattleBusLocation(); /* TODO: Array of Locations where late game could take place in. (POIs) */
+        // GameMode->SafeZonePhase = 1;
+        // GameMode->SafeZoneLocations = Aircraft->K2_GetActorLocation();
+        // Aircraft->K2_GetActorLocation();
+        /* TArray<FVector> Locations;
+        Locations.Add(FVector(-123778, -112480, 17525));
```

Comment:
This was my terrible attempt at custom storms. It does not affect anything in game.